### PR TITLE
Heshan feature ticket

### PR DESCRIPTION
**Pull request by wphkweerasinghe**

### CODE REVIEW

File: PhoenixAirline/src/java/com/phoenixairline/controllers/ticketmanagement/DeleteTicketServlet.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+    // <editor-fold defaultstate="collapsed" desc="HttpServlet methods. Click on the + sign on the left to edit the code.">
+    /**
+     * Handles the HTTP <code>GET</code> method.
+     *
+     * @param request servlet request
+     * @param response servlet response
+     * @throws ServletException if a servlet-specific error occurs
+     * @throws IOException if an I/O error occurs
+     */
```
Comment (this8) [inline PhoenixAirline/src/java/com/phoenixairline/controllers/ticketmanagement/DeleteTicketServlet.java:57]: Remove unwanted comments

File: PhoenixAirline/src/java/com/phoenixairline/controllers/ticketmanagement/DeleteTicketServlet.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+            out.println("<html>");
+            out.println("<head>");
+            out.println("<title>Servlet DeleteTicketServlet</title>");            
+            out.println("</head>");
+            out.println("<body>");
+            out.println("<h1>Servlet DeleteTicketServlet at " + request.getContextPath() + "</h1>");
+            out.println("</body>");
+            out.println("</html>");
+        }
+    }
```
Comment (this8) [inline PhoenixAirline/src/java/com/phoenixairline/controllers/ticketmanagement/DeleteTicketServlet.java:47]: Remove this method as it is not used in our feature

File: PhoenixAirline/src/java/com/phoenixairline/controllers/ticketmanagement/DeleteTicketServlet.java [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    /**
+     * Returns a short description of the servlet.
+     *
+     * @return a String containing servlet description
+     */
+    @Override
+    public String getServletInfo() {
+        return "Short description";
+    }// </editor-fold>
+
```
Comment (this8) [inline PhoenixAirline/src/java/com/phoenixairline/controllers/ticketmanagement/DeleteTicketServlet.java:93]: Remove this method

**Review by this8 — CHANGES_REQUESTED** [review]
Please make the changes.

File: PhoenixAirline/src/java/com/phoenixairline/controllers/ticketmanagement/DeleteTicketServlet.java (hunk 1 above)
Comment (wphkweerasinghe) [reply PhoenixAirline/src/java/com/phoenixairline/controllers/ticketmanagement/DeleteTicketServlet.java:57]: okay done.

File: PhoenixAirline/src/java/com/phoenixairline/controllers/ticketmanagement/DeleteTicketServlet.java (hunk 2 above)
Comment (wphkweerasinghe) [reply PhoenixAirline/src/java/com/phoenixairline/controllers/ticketmanagement/DeleteTicketServlet.java:47]: noted

File: PhoenixAirline/src/java/com/phoenixairline/controllers/ticketmanagement/DeleteTicketServlet.java (hunk 3 above)
Comment (wphkweerasinghe) [reply PhoenixAirline/src/java/com/phoenixairline/controllers/ticketmanagement/DeleteTicketServlet.java:93]: all done

**Pull request merged**